gcc-python-option.c

```diff
@@ -98,7 +98,14 @@ int PyGcc_option_is_enabled(enum opt_code opt_code)
 {
     /* Returns 1 if option OPT_IDX is enabled in OPTS, 0 if it is disabled,
        or -1 if it isn't a simple on-off switch.  */
+#if (GCC_VERSION < 10000)
     int i = option_enabled (opt_code, global_dc->option_state);
+#else
+    /* Starting with GCC 10, options can be distinguished by language. */
+    /* TODO Expose the lang_mask to the user. */
+    int i = option_enabled (opt_code, CL_LANG_ALL, global_dc->option_state);
```

Comment:
Is the ```CL_LANG_ALL``` right here?

Reply:
I believe that that param is used as a mask, so using CL_LANG_ALL here looks correct to me.